src/pages/teams.astro

```diff
@@ -0,0 +1,329 @@
+    question: "Can I invite team members to join Livebook Teams?",
+    answer:
+      "Livebook Teams is not only free but also open source, providing you with a powerful collaborative platform at no cost. We believe in the importance of accessible tools for knowledge sharing and collaboration",
+    open: false,
+  },
+];
+---
+
+<Layout title="Teams" isDarkMode={true}>
+  <section id="intro" class="bg-gray-900 overflow-x-hidden">
+    <div class="m-auto max-w-7xl px-4 py-12">
+      <div class="flex flex-col space-y-10 text-center items-center">
+        <h2 class="max-w-2xl text-4xl font-light text-center text-gray-200">
+          Livebook Teams amplifies Livebook’s capabilities with features
+          designed for
+          <span class="font-bold italic text-brand-pink"
+            >team-based workflows.</span
+          >
+        </h2>
+        <span class="text-lg text-center font-light text-gray-200 max-w-xl">
+          Join our free beta program to gain early access to Livebook Teams and
+          contribute to the development of the roadmap.
+        </span>
+        <a
+          class="px-6 py-4 flex items-center justify-center rounded-xl bg-gray-50 hover:bg-brand-pink text-gray-700 hover:text-gray-100 hover: border hover:border-gray-100 transition duration-150 ease-in-out"
+          href="#create-account"
+        >
+          <Icon icon="arrow-right-line" class="text-xl" />
+          <span class="px-2 font-semibold">Create Account</span>
+        </a>
+        <img
+          class="lg:max-w-5xl rounded-md custom-shadow-1"
+          src={teamsIntroScreenshot}
+          alt="Livebook teams screenshot"
+        />
+      </div>
+    </div>
+  </section>
+  <section id="features" class="bg-gray-800">
+    <div class="m-auto max-w-7xl px-4 py-16 lg:py-20">
+      <div
+        class="flex flex-col space-y-10 sm:space-y-16 text-center items-center"
+      >
+        <div class="m-auto max-w-2xl flex flex-col items-center">
+          <img
+            class="sm:max-w-lg"
+            src={featureOnPremiseNotebook}
+            alt="Feature: On-premise notebook deployment"
+            width="713"
+            height="216"
+          />
+          <h2
+            class="text-gray-50 text-2xl sm:text-4xl lg:text-5xl leading-tight mt-8 sm:mt-10"
+          >
+            On premise notebook <br class="hidden sm:block" />deployment
+          </h2>
+          <span class="text-gray-200 mt-6 sm:mt-8"
+            >Deploy Livebook
+            <div class="group inline-block">
+              <span class="inline-block">apps*</span>
+              <div
+                class="absolute top-[1550px] w-40 p-2 border border-gray-700 bg-gray-900 text-gray-200 text-xs rounded-md opacity-0 group-hover:opacity-100 transition-opacity duration-200 invisible group-hover:visible"
+              >
+                This is a tooltip example.
+              </div>
+            </div> on one or more machines within your infrastructure. Ideal for
```

Comment:
The content is shifted differently on different screen widths, so `top-[1550px]` is not going to be the same. But either way, we already have a tooltip component that you can use :)

```suggestion
            <Tooltip text="TODO tooltip content" direction="top">apps*</Tooltip> on one or more machines within your infrastructure. Ideal for
```

Reply:
ohhhh, ok! =) Thanks, it makes everything easier hehe.